lib/plugin.js

```diff
@@ -2,9 +2,11 @@ import Vue from 'vue'
 import Bugsnag from '@bugsnag/js'
 import BugsnagPluginVue from '@bugsnag/plugin-vue'
 const options = <%= serialize(options) %>
-options.plugins = [new BugsnagPluginVue(Vue)]
-const bugsnagClient = Bugsnag.start(options)
+options.plugins = [new BugsnagPluginVue()]
 
 export default function ({ app }, inject) {
+  const bugsnagClient = Bugsnag.start(options)
+  Bugsnag.getPlugin('vue').installVueErrorHandler(Vue)
```

Comment:
This line was going one line up? Bugsnag instance is created and I get "window is not defined"